src/features/events/store.ts

```diff
@@ -113,6 +113,12 @@ const eventsSlice = createSlice({
         );
       }
     },
+    eventDeselected: (state, action: PayloadAction<number>) => {
```

Comment:
There is already an eventsDeselected, why not use that one?

Reply:
As I wrote in Note to reviewer, it feels weird for me to use `eventsDeselected` because this is not event"s" and other reducers have reducer for "event" and "events". For example, `eventCreated`,`eventsCreated` / `eventLoaded`, `eventsLoaded` / `eventUpdated`, `eventsUpdated` etc... So I followed a same pattern here. But I can use existing `eventsDeselected` reducer! I was just unsure what is the best in this case!